baybe/searchspace/continuous.py

```diff
@@ -301,6 +334,45 @@ def _drop_parameters(self, parameter_names: Collection[str]) -> SubspaceContinuo
             ],
         )
 
+    def _enforce_cardinality_constraints_via_assignment(
```

Comment:
I'm not yet perfectly happy with the name here. Any better suggestions? One alternative I could imagine would be `_enforce_cardinality_constraints_via_bounds`. But input is welcome

Reply:
Since there is only one way of enforcing cardinality consraints, why not simply `enforce_cardinality_constraints`? If people are interested in the details of the how, they can read the docstring.

Reply:
can do, but would keep it private 👍🏼. @Scienfitz: thumbs up is sufficient for approval